src/PostProcessor.cpp

```diff
@@ -591,3 +632,23 @@ FrameBuffer * PostProcessor::doGammaCorrection(FrameBuffer * _pBuffer)
 	_postDraw();
 	return m_pResultBuffer;
 }
+
+FrameBuffer * PostProcessor::doOrientationCorrection(FrameBuffer * _pBuffer)
+{
+	if (_pBuffer == nullptr)
+		return nullptr;
+
+	if (((*REG.VI_STATUS & 8) | config.generalEmulation.enableBlitScreenWorkaround) == 0)
+		return _pBuffer;
```

Comment:
I was following the gamma correction example without fully understanding what that did. Thanks for explaining. I'll fix that.